stable_baselines3/ppo/ppo.py

```diff
@@ -305,17 +301,20 @@ def learn(self,
               n_eval_episodes: int = 5,
               tb_log_name: str = "PPO",
               eval_log_path: Optional[str] = None,
-              reset_num_timesteps: bool = True) -> 'PPO':
+              reset_num_timesteps: bool = True) -> "PPO":
 
         iteration = 0
         callback = self._setup_learn(eval_env, callback, eval_freq,
                                      n_eval_episodes, eval_log_path, reset_num_timesteps)
 
-        # if self.tensorboard_log is not None and SummaryWriter is not None:
-        #     self.tb_writer = SummaryWriter(log_dir=os.path.join(self.tensorboard_log, tb_log_name))
+        utils.configure_logger(self.verbose, self.tensorboard_log, tb_log_name, reset_num_timesteps)
```

Comment:
why not doing that in `setup_learn` ? This is also where `num_timesteps` is reset or not

Reply:
Could you expand? In `configure_logger` I make sure run dir index is kept the same.

Reply:
I meant that you could put `configure_logger` inside `setup_learn`, that would avoid duplicating code